lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -151,22 +155,46 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
       M.select_install_rm_cmd(cache_folder, project_name..'-tmp')
     }
   else
+    local git_folder
+    if is_windows then
+      git_folder = cache_folder ..'\\'.. project_name
+    else
+      git_folder = cache_folder ..'/'.. project_name
+    end
+
+    local clone_error = 'Error during download, please verify your internet connection'
+    if is_windows then
+      clone_error = clone_error .. ". If on Windows you may need to enable Developer mode"
+    end
+
     return {
       {
         cmd = 'git',
         info = 'Downloading...',
-        err = 'Error during download, please verify your internet connection',
+        err = clone_error,
         opts = {
           args = {
             'clone',
+            '-c', 'core.symlinks=true',
```

Comment:
symlinks should not be needed.

Reply:
I think you should remove this an let people handle symlinks the way they want ?
This worked well without this change before I think, could you confirm that it is the case ?